### DISCUSSION TITLE
advised to add a feature

### DISCUSSION THREAD

**Discussion by Nyx2022** [projectdiscovery/retryablehttp-go#24] [Q&A] [answered]
in python,we can set host in headers to the value that different from the real host of http url,however,by my test,[retryablehttp-go](https://github.com/projectdiscovery/retryablehttp-go) may not support this feature,so i advise [retryablehttp-go](https://github.com/projectdiscovery/retryablehttp-go) to support this feature

**Comment by Mzack9999** [comment, marked as answer]
@Nyx2022 This is already supported. You can do it either by setting `request.Host="whatever"` or by setting the header directly:
```console
req, _ := retryablehttp.NewRequest(http.MethodGet,"http://host",nil)
req.Host = "whatever"
req.Header.Set("Host", "whatever")
```
